tests/components/group/test_init.py

```diff
@@ -240,6 +240,29 @@ async def test_expand_entity_ids_ignores_non_strings(hass: HomeAssistant) -> Non
     assert [] == group.expand_entity_ids(hass, [5, True])
 
 
+async def test_expand_entity_ids_domain_group(hass: HomeAssistant) -> None:
+    """Test expand_entity_ids with domain group."""
+    hass.states.async_set("light.Bowl", STATE_ON)
+    hass.states.async_set("light.Ceiling", STATE_OFF)
```

Comment:
I may be wrong, but I think that the project does not allow changing `hass.states` directly anymore, so you should use `hass.services.async_call` to call the `light.turn_on` service